Exercise_D and E/index.html

```diff
@@ -1,22 +1,77 @@
 <!DOCTYPE html>
 <html class="no-js">
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title></title>
-    <meta name="description" content="" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="stylesheet" href="./assets/style/index.css" />
-    <link
-      rel="stylesheet"
-      href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
-      integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh"
-      crossorigin="anonymous"
-    />
-  </head>
-  <body>
-    <!-- CODE HERE -->
-
-    <script src="" async defer></script>
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <title>Flower Shop</title>
+  <meta name="description" content="" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+    integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous" />
+  <link rel="stylesheet" href="./assets/style/index.css" />
+</head>
+
+<body>
+  <!-- CODE HERE -->
+  <header class='logo'>
+    <div class='logo'>
+      <img src="../Exercise_D and E/assets/images/logo.png" alt="">
+    </div>
+  </header>
+
+  <section class="beautifulFlowers">
+    <div class="title">
+      <h1 class="heading">Beautiful Flowers</h1>
+    </div>
+    <div>
+      <button class="learnMore">Learn More</button>
+    </div>
+  </section>
+
+  <section class="info">
+    <div class="row">
+      <div class="col-6">
+        <div>
+          <img src="../Exercise_D and E/assets/images/flower-icon.png" alt="flowerIcon">
```

Comment:
This path should be the relative string that is evaluated from this file.

You therefore should never need to go out of the containing folder.